tools/osbuild-depsolve-dnf

```diff
@@ -54,6 +54,11 @@ class Solver():
 
         for repo in repos:
             self.base.repos.add(self._dnfrepo(repo, self.base.conf))
+
+        if repos_dir:
```

Comment:
This means mixing local dirs and passed in repos is fine? It probably is just want to double check :)

Reply:
It should be, but I'd like to test it.

Reply:
Curious if it will create issues with potential repo ID conflicts.  Maybe we should make them mutually exclusive at first and figure it out later.

Reply:
Mixing actually works (see tests)